src/Digdir.Domain.Dialogporten.Infrastructure/Altinn/Authorization/LocalDevelopmentAltinnAuthorization.cs

```diff
@@ -39,16 +39,16 @@ public LocalDevelopmentAltinnAuthorization(IDialogDbContext db)
         var allParties = dialogData.Select(x => x.Party).Distinct().Take(1000).ToList();
         var allResources = dialogData.Select(x => x.ServiceResource).Distinct().Take(1000).ToList();
 
-        var hasMorePartiesThanResources = allParties.Count > allResources.Count;
         var authorizedResources = new DialogSearchAuthorizationResult
         {
-            PartiesByResources = hasMorePartiesThanResources ? allResources.ToDictionary(resource => resource, _ => allParties) : new(),
-            ResourcesByParties = hasMorePartiesThanResources ? new() : allParties.ToDictionary(party => party, _ => allResources)
+            ResourcesByParties = allParties.ToDictionary(party => party, _ => allResources)
         };
 
         return authorizedResources;
     }
 
-    public async Task<AuthorizedPartiesResult> GetAuthorizedParties(IPartyIdentifier authenticatedParty, CancellationToken cancellationToken = default)
+    [SuppressMessage("Performance", "CA1822:Mark members as static")]
+    [SuppressMessage("Style", "IDE0060:Remove unused parameter")]
+    public async Task<AuthorizedPartiesResult> GetAuthorizedParties(IPartyIdentifier authenticatedParty, bool flatten = false, CancellationToken cancellationToken = default)
```

Comment:
```suggestion
    public async Task<AuthorizedPartiesResult> GetAuthorizedParties(IPartyIdentifier authenticatedParty, bool _, CancellationToken __)
```

Are you getting compiler errors on this one? 🤔 
I saw red squigglies when I first opened the PR, but a re-build removed them .. 
Anyways, we could use the discard param style, and it should shut the analyzer up